### DISCUSSION TITLE
Boot errors

### DISCUSSION THREAD

**Discussion by mvwijland** [thehighnibble/thehighnibble.github.io#86] [Q&A]
I have checked the documentation and the troubleshooting parts. But haven't found the answer just yet. I get the following error:

> rst:0x3 (SW_RESET), boot:0×33 (SPI_FAST_FLASH_BOOT)
> configsip: 188777542, SPIWP: Oxee
> clk_drv:0x00, q_drv: 0x00, d_drv: 0x00, cs0_drv: 0x00, hd_drv: 0x00, w_drv: 0x00
> mode:DIO, clock div:1
> load:0x3fff0018, len:4
> load:0x3fff001c, len: 4492
> load:0×40078000, len:13324
> ho 0 tail 12 room 4
> load: 0×40080400, len: 3796
> entry 0×40080624
> E (61) flash_parts: partition 0 invalid magic number Oxd0aa
> E (62) boot: Failed to verify partition table
> E (63) boot: load partition table error! 

It's the uSD that came with the IMSAI. Anyone has an idea how i can correct this? Help much appreciated.

**Comment by pfcushing** [comment]
I would suggest that maybe something went wrong when the uSD was being initially loaded.  Take it out and insert it into a computer and look the root of the uSD card.  There should only be three folders there, cromemco, imsai and www.  If you have other than that, you should download the latest image file and rebuild the uSD card.

Paul Cushing

**Reply by mvwijland** [reply]
Thx for replying Paul, but it didn't work with a new uSD and the latest software. Apparently the esp is not working correctly, going to re-flash it with the latest binaries. hope that helps.
